docs/excel/custom-functions-debugging.md

```diff
@@ -0,0 +1,90 @@
+---
+ms.date: 02/21/2019
+description: Debug your custom functions in Excel.
+title: Custom functions debugging (preview)
+localization_priority: Normal
+---
+# Custom functions debugging (preview)
+
+Debugging for custom functions can be accomplished by multiple means, depending on what platform you are using. Methods differ between [Office Online for Windows](#windows-10-and-edge), [Office Online for Mac](#mac-and-chrone), Office Desktop for Windows, and Office Desktop for Mac. You can also issue debugging commands through [the command line directly](#using-the-command-line).
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+## Office Online
+### Windows 10 and Edge
+
+1. Open your custom functions root project folder in VS Code.
+2. You may be prompted to install recommended extensions if you have based this project off of the latest yo office generator project. You should select install all, which will install [Debugger for Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) and [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extensions for VS Code. If you are not prompted to install these, install these extensions to VS Code manually.
+3. Select **Terminal | Run Task** and type or select Dev Server.
+4. Switch to Debug View selecting **View | Debug** or by entering **Ctrl+Shift+D**.
+5. From the Debug options, choose Office Online (Edge).
+6. Open Excel Online using Edge and create a new workbook.
```

Comment:
```suggestion
6. Using a Microsoft Edge browser, open Excel Online and create a new workbook.
```

Reply:
"Using **the** Microsoft Edge browser"...?